components/suggested-actions.spec.tsx

```
@@ -0,0 +1,181 @@
+    expect(screen.getByText('Help me write an essay')).toBeInTheDocument();
+    expect(screen.getByText('about silicon valley')).toBeInTheDocument();
+    
+    expect(screen.getByText('What is the weather')).toBeInTheDocument();
+    expect(screen.getByText('in San Francisco?')).toBeInTheDocument();
+  });
+
+  it('should call append with correct message when Next.js button is clicked', async () => {
+    render(<SuggestedActions {...defaultProps} />);
+    
+    const nextjsButton = screen.getByText('What are the advantages').closest('button');
+    fireEvent.click(nextjsButton!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)